examples/podtatohead-deployment/app.yaml

```diff
@@ -19,4 +19,6 @@ spec:
     - name: podtato-head-hat
       version: 0.1.0
   postDeploymentTasks:
-    - post-deployment-hello 
+    - post-deployment-hello
+  preDeploymentEvaluations:
```

Comment:
should we provide a separate example for demonstrating the evaluations? Otherwise, the podtato head example will not work out of the box anymore, unless prometheus is installed in the cluster

Reply:
We could make this part of the observability example, or create a new one ...

Reply:
removed the example from this PR, as it will be part of another PR